flappy-kray/components/flappy-kray/application.yaml

```diff
@@ -20,8 +22,21 @@ spec:
       values: |
         image:
           repository: ghcr.io/konstructio/flappy-kray
+
         ingress:
-          enabled: false
+          enabled: true
```

Comment:
It's the other way around @fharper: because of the location where ingresses are set, both ArgoCD and any other integration fails to connect the dots between the ingress and the deployment. 

The ingress ends up installed on the management cluster even if `flappy-kray` is installed in a different cluster altogether, that's not going to work 😝 

See here for example: https://argocd.mgmt-24.konstruct.io/applications/argocd/mgmt-24-flappy-kray-components

Reply:
Ingresses must go tied to the deployment that they're connected to. If `flappy-kray` needs an ingress, it should deploy it themselves. 

(Not saying I agree with the design, but that's how it works; there's a new Gateway API that supercedes Ingresses we should be using that technically fixes this).

Reply:
I do agree that having the values in a separate values.yaml file is better for readability. But having the `ingress.yaml` defined at the same level as the Argo CD application deploys it to the management cluster instead of the the destination cluster. ( i.e when installting flappy-kray in a downstream account the ingress is applied in management cluster. @CristhianF7 and @patrickdappollonio discovered this during development)

Reply:
I didn't see that you commented when I was writing. Can you ELi5 how GAPI will solve this?

Reply:
Seems like it's the way to go from now on. Did you test the app to be sure it's working properly like this?

Reply:
I didn't see anyone's comment while writing my last one.